network/p2p/peerManager.go

```diff
@@ -118,15 +118,8 @@ func (pm *PeerManager) updatePeers() {
 
 	// ask the connector to connect to all peers in the list
 	err = pm.connector.UpdatePeers(pm.unit.Ctx(), ids)
-	if err == nil {
-		return
-	}
-
-	if IsUnconvertibleIdentitiesError(err) {
-		// log conversion error as fatal since it indicates a bad identity table
-		pm.logger.Fatal().Err(err).Msg("failed to connect to peers")
-		return
+	if err != nil {
+		// one of more identities in the identity table could not be connected to
+		pm.logger.Error().Err(err).Msg("failed to connect to one or more peers")
```

Comment:
It seems this can only be an `NewUnconvertableIdentitiesError` in our implementation. Could we unpack this case and thereby maybe get a more usable error message?

Reply:
so `UnconvertableIdentitiesError` is [logged as multiple ](https://github.com/onflow/flow-go/blob/187f08c4724d84f6d77e5de49868fe22842b6b18/network/p2p/libp2pConnector.go#L40-#L46)individual errors (multi-error) which will list out the Identity and the error reason.
Will that suffice?

Reply:
yep, all good!